security/us-east-1/security-hub/README.md

```diff
@@ -0,0 +1,46 @@
+# AWS Security Hub Configuration
+
+This directory contains the Terraform configuration files for setting up AWS Security Hub in the `us-east-1` region.
```

Comment:
I wonder if the reference you made to the `us-east-1` region is really useful, since this layer is already under the `us-east-1` subdirectory. Or maybe you intended to communicate something else that I'm not getting? 🤔

Reply:
ok fixed